SherpaBackEnd/SurveyNotification/Infrastructure/Http/SurveyNotificationController.cs

```diff
@@ -0,0 +1,45 @@
+using Microsoft.AspNetCore.Mvc;
+using SherpaBackEnd.Exceptions;
+using SherpaBackEnd.SurveyNotification.Application;
+using SherpaBackEnd.SurveyNotification.Infrastructure.Http.Dto;
+
+namespace SherpaBackEnd.SurveyNotification.Infrastructure.Http;
+
+[ApiController]
+[Route("survey-notifications")]
+public class SurveyNotificationController
+{
+    private readonly ISurveyNotificationService _surveyNotificationService;
+    private readonly ILogger<SurveyNotificationController> _logger;
+
+    public SurveyNotificationController(ISurveyNotificationService surveyNotificationService,
+        ILogger<SurveyNotificationController> logger)
+    {
+        _surveyNotificationService = surveyNotificationService;
+        _logger = logger;
+    }
+
+    [HttpPost]
+    public async Task<IActionResult> LaunchSurvey(CreateSurveyNotificationsDto createSurveyNotificationsDto)
```

Comment:
I'm not a fan of the naming now that I look at it in review since this is the survey notification controller, I understand it's the way the business says it but it seems like it isn't aligned with the controller